### ISSUE TITLE
No-Kernal Image Error when using Qulacs-GPU

### ISSUE THREAD

**Issue by irfankhan10** [qulacs/qulacs#275]
I get the following error when using Qulacs-GPU installed via PyPI:

`Error: /tmp/pip-install-xt7zsonc/qulacs-gpu_87dd692bc8aa4961aa66cf205ec58776/src/gpusim/memory_ops.cu:71, code: 209, reason: no kernel image is available for execution on the device`

CUDA Version: 11.1
Driver Version: 455.45.01
NVIDIA-SMI: 455.45.01
NVIDIA GPU Tesla K80

Please can you advise.

Googling "CUDA code: 209" suggests it is an issue with Qulacs, but I am not 100% sure.

**Comment by corryvrequan** [comment]
Sorry for late reply. @irfankhan10 .
I'm also not sure why it happens since I haven't seen this error message. As far as we tested with Tesla V100 or recent GTX1080 and CUDA Version 10, Qulacs can be built successfully and passes tests. As far as I googled, I understand that this error suggests that CUDA compiler outputs a binary that is not supported by Tesla K80 architecture.

While I'm not sure, I guess, a possible reason is that the CUDA computing compatibility of Tesla K80 is not enough for building Qulacs with CUDA 11.1. 
So, downgrading CUDA11 to CUDA10 may resolve the problem, but I guess downgrading drivers may affect the environment and I cannot guarantee qulacs works with it...
